src/app/containers/Navigation/index.canonical.test.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { render, fireEvent } from '@testing-library/react';
+import Navigation from './index.canonical';
+
+const scrollableTestId = 'scrollable-list';
+const dropdownTestId = 'dropdown';
+
+const scrollableListItems = (
+  <ul data-testid={scrollableTestId}>
+    <li>List Items</li>
+  </ul>
+);
+
+const dropdownListItems = <div data-testid={dropdownTestId}>Dropdown</div>;
+
+const navigation = (
+  <Navigation
+    scrollableListItems={scrollableListItems}
+    dropdownListItems={dropdownListItems}
+    skipLinkText="skip link"
+    menuAnnouncedText="menu"
+    script={latin}
+    service="news"
+    dir="ltr"
+  />
+);
+
```

Comment:
What if we put `scrollableTestId`, `dropdownTestId`, `scrollableListItems`, `dropdownListItems` and `navigation` in a file, and imported them here and in `src/app/containers/Navigation/index.amp.test.jsx` - as they are mostly similar?